mlrun/api/main.py

```diff
@@ -160,6 +162,7 @@ async def move_api_to_online():
             _start_periodic_cleanup()
             _start_periodic_runs_monitoring()
             await _start_logs_collection()
+            await _start_periodic_stop_logs()
```

Comment:
I think that we should make sure that if one function fails it won't hurt the other from running